tests/helpers.py

```diff
@@ -0,0 +1,44 @@
+"""
+Helper functions for OCS tests
+"""
+import os
+import logging
+import ocs.ocp
+import ocs.defaults as defaults
+from utility import templating
+
+logger = logging.getLogger(__name__)
+
+
+OCP = ocs.ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
```

Comment:
namespace in this case is defined in two places. 

1) namespace=defaults.ROOK_CLUSTER_NAMESPACE
2) PROJECT_NAME = 'test-project'

I think this is be problematic. I'd not define the namespace here, rather define it like how you have done in line 22 and use that value throughout the test. Also, by doing that we use the same OCP object for different namespaces if required. This gives more flexibility while writing test case instead of going with a pre-defined name in config file. 

p.s., This comment is applicable for all the places where namespace is defined in ocs.ocp.OCP object.

Reply:
Done